scripts/maintenance/generate_audit_summary.py

```
@@ -0,0 +1,94 @@
+        'NPMRC': 0,
+        'ROADMAP': 0,
+        'GVC': 0,
+        'REVIEW': 0,
+        'HELPERS': 0,
+        'UNREADABLE': 0
+    }
+
+    for item in data.get('blocked', []):
+        issues = item['issues']
+        issues_str = str(issues)
+        if 'Orphan branch' in issues_str: stats['Orphan'] += 1
```

Ruff (0.15.10)

[error] Multiple statements on one line (colon)

(E701)